draft-pauly-ohai-svcb-config.md

```diff
@@ -168,34 +180,42 @@ still need to ensure that they are not being targeted with unique
 key configurations that would reveal their identity. See {{security}} for
 more discussion.
 
-### Handling Oblivious DoH Configurations
+# Configuration Well-Known URI
+
+Once a client has discovered that a service is available as an oblivious
+target, it needs to know the key configuration before sending oblivious
+requests.
+
+This document defines a well-known URI {{!RFC8615}}, "oblivious-configs",
+that allows a target to host its configurations.
 
-Oblivious DoH was originally defined in
-{{?ODOH=I-D.draft-pauly-dprive-oblivious-doh}}. This version of
-Oblivious DoH uses a different key configuration format than
-generic Oblivious HTTP. SVCB records using the "dns" scheme
-can include one or more `ObliviousDoHConfig` structures 
-using the "odoh-configs" parameter.
+The URI is constructed using the TargetName in the SVCB record.
 
-In wire format, the value of the "odoh-configs" parameter is one
-or more `ObliviousDoHConfigs` structures {{ODOH}} concatenated
-together. In presentation format, the value is the same structures
-encoded in Base64 {{!BASE64=RFC4648}}.
+For example, the URI for the following record:
 
-All other requirements for "ohttp-configs" in this document apply
-to "odoh-configs".
+~~~
+svc.example.com. 7200  IN HTTPS 1 . alpn=h2,h2 oblivious
+~~~
 
-# Deployment Considerations
+would be "https://svc.example.com/.well-known/oblivious-configs".
 
-Deployments that add the "ohttp-configs" SvcParamKey need to be
-careful to add this only to services meant to be accessed using
-Oblivious HTTP. Information in a single SVCB record that contains
-"ohttp-configs" only applies to the oblivious service, not
-other HTTP services.
+As another example, the URI for the following record:
 
-If a service offers both traditional HTTP and oblivious HTTP, these can
-be represented by separate SVCB or HTTPS records, both with and
-without the "ohttp-configs" SvcParamKey.
+~~~
+_dns.resolver.arpa  7200  IN SVCB 1 doh.example.net (
+     alpn=h2 dohpath=/dns-query{?dns} oblivious )
+~~~
+
+would be "https://doh.example.net/.well-known/oblivious-configs".
+
+The content of this resource is expected to be "application/ohttp-keys",
+as defined in {{OHTTP}}.
+
+Before being able to use a server as an oblivious target, clients need
+to use this URI to fetch the configuration. They can either fetch it
+directly, or do so via a proxy in order to avoid the server discovering
+information about the client's identity. See {{security}} for more
+discussion of avoiding key targeting attacks.
 
 # Security and Privacy Considerations {#security}
```

Comment:
The proposed mechanism is susceptible to downgrade attack not to use oblivious HTTP. I mean an attacker removing the oblivious SvcParamKey from the DNS response. Couple of possible ways to mitigate the attack could be (1) to perform the DNS SVCB query again after secure transport connection is established with the discovered DNS resolver (2) Rely on DNSSEC.

Reply:
Added text in the security considerations section!